apps/masterbots.ai/components/shared/external-link.tsx

```
@@ -1,26 +1,26 @@
   href,
+  className,
   children
 }: {
   href: string
+  className: string
   children: React.ReactNode
 }) {
   return (
     <a
       href={href}
       target="_blank"
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)